docs/source/blogs/Best_perf_practice_on_DeepSeek-R1_in_TensorRT-LLM.md

```diff
@@ -138,7 +138,8 @@ YOUR_DATA_PATH=<your dataset file following the format>
 
 cat >./extra-llm-api-config.yml<<EOF
 use_cuda_graph: true
-moe_backend: TRTLLM
+moe_config:
```

Comment:
Blocking the merge by leaving a thread.

Reply:
It should be fine to proceed according to the email thread.